.github/workflows/ci.yml

```
@@ -0,0 +1,340 @@
+          ~/.cargo/git
+          target
+        key: perf-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+    
+    - name: Run performance regression tests
+      run: |
+        # Test key performance metrics
+        cargo test --release --test integration_tests test_large_buffer_performance -- --ignored
+        cargo test --release --test integration_tests test_extensive_text_writing -- --ignored
+        cargo test --release --test integration_tests test_rapid_text_input_stress -- --ignored
+        cargo test --release --test integration_tests test_frequent_resize_stress -- --ignored
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)